### DISCUSSION TITLE
[POSITION FILLED] Red Hat's Hiring: Observability & Quarkus, Software Engineering Role

### DISCUSSION THREAD

**Discussion by paulrobinson** [quarkusio/quarkus#22196] [Jobs and Opportunities]
**POSITION NOW FILLED**

Hi All,

We have a job opening for a core engineer on the Quarkus team!

We’re looking for someone to focus on Observability capabilities in Quarkus, but also to some extent Red Hat’s other runtimes. They will be continuing to develop our Open Telementy implementations, whilst also enhancing our integration with Observability SaaS providers (such as Honeycomb, Datadog, Stackdriver, etc).  We also need someone that can work across Red Hat’s other Runtimes (such as WildFly and JBoss EAP) to develop our observability strategy and ensure consistency.

The position is open to remote candidates in Europe.

Here’s the job listing: [https://global-redhat.icims.com/jobs/92056/](https://global-redhat.icims.com/jobs/92056/principal-software-engineer-%28application-services-observability%29---remote-working/job?hub=7&mobile=false&width=1140&height=500&bga=true&needsRedirect=false&jan1offset=0&jun1offset=60)

Thanks,

Paul.

**Comment by bentito** [comment]
Hi @paulrobinson - this job listing isn't up any more.

**Comment by paulrobinson** [comment]
Hi @bentito, this position is now filled. My apologies, I forgot to report back here.
